subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/transform/ExecuteScheduledTransformBuildOperationType.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.api.internal.artifacts.transform;
+
+import org.gradle.internal.operations.BuildOperationType;
+import org.gradle.internal.scan.NotUsedByScanPlugin;
+
+/**
+ * @since 5.1
+ */
+@NotUsedByScanPlugin
+public class ExecuteScheduledTransformBuildOperationType implements BuildOperationType<ExecuteScheduledTransformBuildOperationType.Details, ExecuteScheduledTransformBuildOperationType.Result> {
```

Comment:
Let's call this thing `Transformation` instead of `Transform`, shall we? There is the `Transformer` which does the `Transformation`.

Moreover, I think it is a `ScheduledTransformationStep` as opposed the the `Transformation` which only transforms a single file and not a whole subject which ma consist of several files.

Reply:
Changed in b7db7106df0.